package/firmware/omnia-mcu-firmware/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2023 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-mcu-firmware
+PKG_DISTNAME:=omnia_hw_ctrl
+PKG_VERSION:=3.4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_SUBDIR:=$(PKG_DISTNAME)-binaries-v$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.bz2
+PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/hw/$(PKG_DISTNAME)/-/releases/v$(PKG_VERSION)/downloads/
+PKG_HASH:=2b0888fb065b279dc5eac90a76fea8e2353bd554b16df07515c4114d6d0a2c8e
+
+PKG_MAINTAINER:=Marek Mojik <marek.mojik@nic.cz>
+PKG_LICENSE:=GPL-3.0-or-later
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_SUBDIR)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/omnia-mcu-firmware
+  SECTION:=firmware
+  CATEGORY:=Firmware
+  URL:=$(PKG_SOURCE_URL)
+  TITLE:=CZ.NIC Turris Omnia MCU firmware
+  DEPENDS:=@TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia
+endef
+
+define Package/omnia-mcu-firmware/description
+Firmware binaries for the microcontroller on the Turris Omnia router. These are
+used by the omnia-mcutool utility when upgrading MCU firmware.
+endef
+
+define Build/Compile
+	true
+endef
+
+define Package/omnia-mcu-firmware/install
+	$(INSTALL_DIR) $(1)/usr/share/omnia-mcu-firmware/
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/*.bin $(1)/usr/share/omnia-mcu-firmware/
```

Comment:
Can not we specify which binaries we will have in the package instead of having all binaries, there?

Reply:
I don't know, I have specifically created the tarball (`omnia_hw_ctrl-binaries-v3.4.tar.bz2`) to contain only the files that need to be installed, so that I would not need to manually name the 10 files in the package Makefile :)

Grepping through the tree, there are several packages using file name expansions in install macros, `git grep 'INSTALL.*/\*'` matches 64 lines in the repository.